src/ru/nsu/ccfit/zuev/osu/game/Spinner.java

```diff
@@ -290,8 +286,8 @@ public void update(final float dt) {
         if (percentfill > 1 || clear) {
             percentfill = 1;
             if (!clear) {
-                clearText = SpritePool.getInstance().getCenteredSprite(
-                        "spinner-clear", new PointF(center.x, center.y * 0.5f));
+                final PointF pos1 = new PointF(center.x, center.y * 0.5f);
+                clearText = new CentredSprite(pos1.x, pos1.y, ResourceManager.getInstance().getTexture("spinner-clear"));
```

Comment:
Same as above

Reply:
`clearText` instantiaton was moved to class constructor.